engine/index.js

```diff
@@ -421,13 +421,55 @@ handlebars.registerHelper('if_eq', function comparison(left, right, opts) { // N
   return opts.inverse(this);
 });
 
+// status partial
+const featureStatusContents = fs.readFileSync('src/tpl/featureStatusPartial.html', {
+  encoding: 'utf-8',
+});
+// links partial is needed as the feature page is build in slightly different
+// way (no summary), we might also switch off links when embedded
+const featureLinksContents = fs.readFileSync('src/tpl/featureLinksPartial.html', {
+  encoding: 'utf-8',
+});
+
 function buildIndex(status) {
   const templateContents = fs.readFileSync('src/tpl/index.html', {
     encoding: 'utf-8',
   });
+  handlebars.registerHelper('featureStatusName', function featureStatusName() {
+    return this.slug + '-status';
+  });
+  handlebars.registerHelper('featureLinksName', function featureLinksName() {
+    return this.slug + '-links';
+  });
+  status.features.forEach((featureData) => {
+    // register partials for each feature
+    handlebars.registerPartial(
+        featureData.slug + '-status',
+        handlebars.compile(featureStatusContents)(featureData));
+    handlebars.registerPartial(
+        featureData.slug + '-links',
+        handlebars.compile(featureLinksContents)(featureData));
+  });
   return Promise.resolve(handlebars.compile(templateContents)(status));
```

Comment:
Do we need to wait for handlebars to compile the partials before compiling the template?